### DISCUSSION TITLE
Edit capability

### DISCUSSION THREAD

**Discussion by ks1v** [Freika/dawarich#51] [Ideas]
Recently Google killed Timeline Web -- fairly decent editor for location data -- and left us with hard-to-use timeline in mobile app. Retrospective editing became almost impossible. It would be great to be able to edit data in Dawarich.

Basically it implies this methods: 
1. Nodes: change location or/and time interval
2. Movement: change mode and time interval
3. Nicest one: span to roads over OSM road graph

**Comment by Freika** [comment]
Hi @ks1v!

You mean, you want to be able to edit each point? That might be cumbersome but if it's a common user scenario, why not, I can work on that, too.

About movement I didn't really understood, can you provide an example?

Span roads: you probably mean something like this? https://github.com/Freika/dawarich/discussions/43

**Comment by chrisl8** [comment]
I just want to add my nifty screenshot here, as this is the request I was directed to.

All I want to do is be able to "click" on that one point, and delete it.

![image](https://github.com/user-attachments/assets/2783258b-b4d6-4ce0-8ea1-20e8f2a707ff)

In my mind, I imagine an optional map that displays just points, not the line, and I could select and delete any of them. Something super simple to start with anyway.

**Reply by Freika** [reply]
@chrisl8 actually you can choose what to show here:

<img width="562" alt="изображение" src="https://github.com/user-attachments/assets/46b2a5f4-c220-4287-9b8a-11fdc78bc4fa">

<img width="607" alt="изображение" src="https://github.com/user-attachments/assets/d6be23ce-8b5d-4cea-b4ad-582a5b830ef3">

You can click on any point to get some info on it, and later I'll add possibility to remove it as well

**Comment by chrisl8** [comment]
Thanks, yeah, I can see the points, and with the detailed data, I can jump over to the "points" list and search for it and delete it.

I was just hoping for the ability to delete it from the map instead of having to search for it and delete it from the other screen.

Awesome tool by the way! Thanks!

**Comment by Freika** [comment]
Deleting a single point from a popup was implemented in https://github.com/Freika/dawarich/releases/tag/0.9.4

**Comment by dunc1ca** [comment]
I agree.  It would be nice to edit points directly from the map.  Adding points by clicking on the map.  Editing points by drag and drop and editing the date/time.

**Reply by Freika** [reply]
> Editing points by drag and drop and editing the date/time.

In its current state, Dawarich is meant to operate with existing geodata, not editing or adding new geodata. Deleting functionality is useful for removing points that were tracked mistakenly but adding and moving points is out of the scope of the app and besides there are plenty of other tools to do that already.

**Comment by Freika** [comment]
Points movement added in https://github.com/Freika/dawarich/releases/tag/0.22.4
